etc/docker/travis/Dockerfile

```diff
@@ -32,6 +32,18 @@ RUN yum install -y python-pip gmp-devel krb5-devel httpd mod_ssl mod_auth_kerb m
 RUN yum install -y python34 python34-pip python34-devel
 RUN rm -rf /usr/lib/python2.7/site-packages/ipaddress*
 
+# Install sqlite3 version 3.28
+RUN curl https://www.sqlite.org/2019/sqlite-autoconf-3280000.tar.gz > sqlite.tar.gz
```

Comment:
Just a comment: I am getting a bit concerned about all the different dependency versions which no exists outside of the `pip-requires*` files. Now they are in different dockerfiles, and also in the setup.py of the package when doing someting like `pip install rucio[mysql]`. I think we should have a central way to define all these dependencies and just pull them in from there wherever needed. Not sure though what's the best way to do this